models/db/score.model.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+module.exports = (sequelize, DataTypes) => {
+  const Score = sequelize.define('Score', {
+    uri: {
+      type: DataTypes.STRING,
+      get () {
+        return this.getDataValue('uri').toLowerCase()
+      },
+      set (val) {
+        this.setDataValue('uri', val.toLowerCase())
```

Comment:
I'm not sure if the setter is called before or after validations. You may need to check for null or undefined value before calling toLowerCase()